app/avatar/utils.py

```diff
@@ -148,7 +148,7 @@ def get_avatar_context():
                 ['Masks-guy-fawkes'], ['Masks-jack-o-lantern-lighted'], ['Masks-wolverine_inspired'],
                 ['Masks-captain_inspired'], ['Extras-Parrot'], ['Extras-wonderwoman_inspired'],
                 ['Extras-santa_inspired'], ['Extras-reindeer'], ['Masks-gitcoinbot'],
-                ['Masks-batman_inspired'], ['Masks-eye-patch'], ['Masks-darth_inspired'],
+                ['Extras-tattoo'], ['Masks-batman_inspired'], ['Masks-eye-patch'],
```

Comment:
@nasehim7 why was `Masks-darth_inspired` removed ?

Reply:
@thelostone-mc looks like I messed up with my branch :/

Reply:
I better be removing this inconsistency by providing a quick fix just wait

Reply:
https://github.com/gitcoinco/web/pull/3266

Reply:
Ah thanks man :)